### PR TITLE
Set release date, sort changelog entries and update bug list.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+ * Code Generation: Implement copying recursive structs from storage to memory.
  * Yul: Disallow function definitions inside for-loop init blocks.
 
 
 Compiler Features:
  * ABI Decoder: Raise a runtime error on dirty inputs when using the experimental decoder.
- * Standartd JSON Interface: Metadata settings now re-produce the original 'useLiteralContent' setting from the compilation input.
+ * Standartd JSON Interface: Metadata settings now re-produce the original ``"useLiteralContent"`` setting from the compilation input.
+ * Optimizer: Add rule for shifts by constants larger than 255 for Constantinople.
+ * Optimizer: Add rule to simplify certain ANDs and SHL combinations
```
Comment (bshastry) [inline Changelog.md:17]: Nit: I noticed that the bullets under Important bugfixes and Language features are sorted but these are not

**Review by bshastry — DISMISSED** [review]
LGTM. Added minor comment about sorting changelog bullet points under compiler features.

File: Changelog.md [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
- * Code Generation: Implement copying recursive structs from storage to memory.
  * ABIEncoderV2: Implement encoding of calldata arrays and structs.
+ * Code Generation: Implement copying recursive structs from storage to memory.
  * Yul: Disallow function definitions inside for-loop init blocks.
 
 
 Compiler Features:
  * ABI Decoder: Raise a runtime error on dirty inputs when using the experimental decoder.
- * Standartd JSON Interface: Metadata settings now re-produce the original 'useLiteralContent' setting from the compilation input.
+ * Standartd JSON Interface: Metadata settings now re-produce the original ``"useLiteralContent"`` setting from the compilation input.
```
Comment (leonardoalt) [inline Changelog.md]: Not in alphabetical order

Comment (chriseth) [reply Changelog.md]: Furthermore, it contains a really bad typo. :)

**Comment by chriseth** [comment]
Better now?

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**